tests/tools/fio/options.go

```diff
@@ -0,0 +1,20 @@
+package fio
+
+// Options are flags to be set when running fio
+type Options map[string]string
+
+// Merge will merge two Options, overwriting common option keys
+// with the incoming option values. Returns the merged result
+func (opt Options) Merge(otherOpt Options) map[string]string {
+	out := make(map[string]string)
```

Comment:
The resulting map, will have at least the maximum of `len(opt)` and `len(otherOpt)` and it could be as large as len(opt) + len(otherOpt)`. Not sure if it makes any difference to take advantage of that or not.

Reply:
Gave a capacity hint in the make function as the sum of the two lengths